Source/WTF/wtf/text/Base64.h

```diff
@@ -34,30 +34,27 @@
 
 namespace WTF {
 
-enum class Base64EncodePolicy {
+enum class Base64EncodePolicy : bool {
     DoNotInsertLFs,
-    InsertLFs,
     URL // No padding, no LFs.
 };
 
-enum class Base64EncodeMap { Default, URL };
+enum class Base64EncodeMap : bool { Default, URL };
 
 enum class Base64DecodeOptions : uint8_t {
     ValidatePadding         = 1 << 0,
     IgnoreSpacesAndNewLines = 1 << 1,
     DiscardVerticalTab      = 1 << 2,
 };
 
-enum class Base64DecodeMap { Default, URL };
+enum class Base64DecodeMap : bool { Default, URL };
```

Comment:
is `bool` better than `uint8_t`? IIRC best practice was to use uint8_t because of some quirk of MSVC

Reply:
I modeled this after some internal discussion and commits such as cebd0f9727d7.